backend/pong/game_logic.py

```diff
@@ -30,9 +30,9 @@ def update_ball_position(self, pong_stat):
 
     def check_collisions(self, pong_stat):
         # Check for collisions with players
-        if pong_stat.ball.x <= pong_stat.player_left.x + PLAYER_WIDTH and pong_stat.player_left.y <= pong_stat.ball.y <= pong_stat.player_left.y + PLAYER_HEIGHT:
+        if pong_stat.ball.x <= pong_stat.player_left.x + PLAYER_WIDTH and pong_stat.player_left.y - PLAYER_HEIGHT / 2 <= pong_stat.ball.y <= pong_stat.player_left.y + PLAYER_HEIGHT / 2:
```

Comment:
smallest thing but still, as we know we always want to check PLAYER_HEIGHT / 2, we could just create a constant of that one too.

Reply:
Also taken care of 😂